src/components/tiles/location-tile.tsx

```diff
@@ -0,0 +1,55 @@
+import { FaWaze } from 'react-icons/fa';
+import { SiGooglemaps } from 'react-icons/si';
+
+import { Tile } from './tile';
+
+const mapLinks = {
+  google: 'https://goo.gl/maps/dkXw1AjgX351WY4J6',
+  waze: 'https://ul.waze.com/ul?preview_venue_id=12517851.124916363.422156&navigate=yes',
+};
+
+export function LocationTile() {
+  return (
+    <Tile className='col-span-3 min-h-[500px]'>
```

Comment:
This has to be `sm:col-span-2 lg:col-span-3` 
This is the class of the container: `grid grid-cols-1 sm:grid-cols-2 lg:grid-cols-3` So there is 1 column on base width, 2 on small, 3 above lg. If an item takes up more columns than there are, it messes up the other items